requirements.txt

```diff
@@ -93,26 +93,28 @@ packaging==24.0
     # via gunicorn
 pillow==10.3.0
     # via -r requirements.in
-pydantic==2.7.0
+pydantic==2.6.0
     # via
     #   -r requirements.in
     #   covjson-pydantic
     #   edr-pydantic
     #   fastapi
     #   geojson-pydantic
-pydantic-core==2.18.1
+pydantic-core==2.16.1
     # via pydantic
 python-dateutil==2.9.0.post0
-    # via owslib
+    # via
+    #   -r requirements.in
+    #   owslib
 pytz==2023.4
     # via
     #   -r requirements.in
     #   owslib
 pyyaml==6.0.1
     # via owslib
-redis==5.0.3
+redis==5.0.4
     # via -r requirements.in
-requests==2.31.0
+requests==2.32.2
```

Comment:
Note: the updated packages from `requirements.txt`, besides the `edr-pydantic` package, are already updated in master. This was done today in: https://github.com/KNMI/adaguc-server/pull/370/files.

I think you can merge master in your branch to solve this (maybe github resolves this automatically? I'm not sure).

Reply:
Done